docs/content/eventing/_index.md

```diff
@@ -1,17 +1,143 @@
 
 <!--more-->
 
-<!-- replace this comment with the content of the article -->
\ No newline at end of file
+## Glossary
+
+Here's a handy list of terms to help get you started:
+
+#### **api key** <a name="api-key"></a>
+"API" stands for "Application Programming Interface", which is a very broad term that refers (super high level) to the ways in which users or other applications can interact with an application.
+
+Some applications (like Ensign) require permission to interact with, such as a password, token, or key.
+
+You can get a free Ensign API key by visiting rotational.io/ensign. Your key will consist of two parts, a `ClientID` and a `ClientSecret`. The `ClientID` uniquely identifies you, and the `ClientSecret` proves that you have permission to create and access event data. You will need to pass both of these in to create an Ensign [client](#client) connection.
+
+#### **asynchronous** <a name="asynchronous"></a>
+An asynchronous microservice is one in which requests to a service and the subsequent responses are decoupled and can occur independently of each other.
+
+This differs from the **synchronous** pattern, in which a client request (e.g. a query) is blocked from moving forward until a server response is received. Synchronous microservices can result in cascading failures and compounding latencies in applications.
+
+Asynchronous microservices can make it a lot easier for teams to develop and deploy components independently.
+
+Asynchronous microservices require an intermediary service usually known as a [broker](#broker) to hold messages emitted by a publisher that are awaiting retrieval from subscribers.
+
+#### **broker** <a name="broker"></a>
+An event broker is an intermediary service inside an asynchronous eventing system that stores events sent by publishers until they are received by all subscribers.
+
+Brokers are also in charge of things like keeping events in the correct order, remembering which subscribers are listening to a topic stream, recording the last message each subscriber retrieved, etc.
+
+In Ensign, brokers can save events permanently even after they have been retrieved (to support "time travel" &mdash; the ability to retroactively scan through an event stream to support analytics and machine learning).
+
+#### **client** <a name="client"></a>
+In order to write or read data from an underlying data system (like a database or event stream), you need a client to connect to the data system and interact with it as needed (such as reading and writing data). This connection often looks something like `conn = DBConnection(credentials)`, and after creating the `conn` variable, subsequent lines of code can leverage it to perform the kinds of data interactions you wish to make.
+
+In Ensign, to establish a client, you need an [API key](#api-key).
+
+```golang
+import 	(
+    ensign "github.com/rotationalio/ensign/sdks/go"
+)
+
+client, _ := ensign.New(&ensign.Options{
```

Comment:
I wonder if we should discourage "throwing the error to the underscore" and define an `err` here as well. I don't think we have to have error handling code here because it's nice to advertise the simplicity but I do think it's good to at least hint what the other thing being returned is.

Additionally if someone is copying the code from here and copies in the wrong key or something, an error is way more helpful than a nil panic.